### PR TITLE
Week 6 stripe

### PR DESCRIPTION
**Pull request by jepetko**
- 13    Assignment: Use Stripe Checkout to charge credit card
- 15    Assignment: Use Stripe custom form to charge credit card

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -17,6 +17,13 @@ def create
 
     if @user.save
 
+      begin
+        Stripe::Charge.create(card: params[:stripeToken], amount: 999, description: "Myflix charge for #{@user.email}", currency: 'usd') unless Rails.env.test?
```
Comment (brandoncc) [inline app/controllers/users_controller.rb]: These types of checks should not be in your app code, generally your initializers should take care of whatever setup makes your code okay for all environments.

Comment (brandoncc) [inline app/controllers/users_controller.rb]: I'm talking about the env check in case I wasn't clear.

Comment (jepetko) [inline app/controllers/users_controller.rb]: okay, what is the proper way to use it in the test environment? Something like this?

```
Stripe::Charge.create(card: params[:stripeToken], amount: 999, description: "Myflix charge for #{@user.email}", currency: 'usd') 
if params[:stripeToken]
```

Comment (jepetko) [inline app/controllers/users_controller.rb]: actually I could do this too. The charge should be created only for if there is Stripe configuration.

```
Stripe::Charge.create(card: params[:stripeToken], amount: 999, description: "Myflix charge for #{@user.email}", currency: 'usd') 
if Stripe.api_key
```

Comment (brandoncc) [inline app/controllers/users_controller.rb]: You will be learning about using a gem called vcr in the next lesson which will record http request/response cycles. You should actually be letting stripe get hit in test mode too.

**Pull request merged**